packages/scripts/README.md

```diff
@@ -207,8 +213,38 @@ _Example:_
 ```
 
 This is how you execute those scripts using the presented setup:
+
 * `npm run test:unit` - runs all unit tests.
 * `npm run test:unit:help` - prints all available options to configure unit tests runner.
 * `npm run test:unit:watch` - runs all unit tests in the watch mode.
 
+## Webpack config
```

Comment:
I think it will be best if we move this section to its own `README` file inside `config` folder to make it less prominent. We should also warn that using your own config means you are opting out from easy update path whenever `@wordpress/scripts` gets updated.

Reply:
> I think it will be best if we move this section to its own README file inside config folder to make it less prominent.

mmm, I'm not convinced about this. For people to use the `build`/`start` commands, they need to know some things (the default entry input, the externals provided, and so). We need to document this.

> We should also warn that using your own config means you are opting out from easy update path whenever @wordpress/scripts gets updated.

If this happens, wouldn't this be considered an API change so documented in the CHANGELOG? I'd be fine either as it this or adding a note at the end of the section in this vein:

_Note that this isn't guaranteed to be backward-compatible in future releases if we needed to do some changes. When updating, you should check for changes in the CHANGELOG._

Reply:
> mmm, I'm not convinced about this. For people to use the build/start commands, they need to know some things (the default entry input, the externals provided, and so). We need to document this.

Yes, we should explain what an entry point is and where the code is going to be outputted. I wouldn't discuss `externals` for simplicity. We should reference the detailed description of the config for those who want to learn more.

Reply:
> If this happens, wouldn't this be considered an API change so documented in the CHANGELOG? I'd be fine either as it this or adding a note at the end of the section in this vein:

Well, we will see. My point is, using the default config files makes sure we are taking care of making it backward compatible. Otherwise, they need to study changelog files to learn how to fix.